tests/loaders/test_cb.py

```diff
@@ -9,10 +9,13 @@
 
 @pytest.fixture
 def mock_cbc_sdk(monkeypatch: pytest.MonkeyPatch) -> Iterator[MagicMock]:
-    try:
-        del sys.modules["dissect.target.loader"]
-    except KeyError:
-        pass
+    # The references to cbc_sdk properties in the cb loader will point to the
+    # MagicMock created for the first test function that runs.
+    # Thus we need to delete the cb module to force it to reload when importing
+    # from it so the MagickMock used in the cb module will be the one created
+    # for the test function that is running.
+    if "dissect.target.loaders.cb" in sys.modules:
+        del sys.modules["dissect.target.loaders.cb"]
```

Comment:
Would this work/be better? https://docs.pytest.org/en/latest/reference/reference.html#pytest.MonkeyPatch.delitem

Reply:
Yes, that's even nicer